recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - pip
     - python
     - hatchling >=1.4
```

Comment:
is this version range here really good?  we should avoid version ranges in host section

Reply:
Done!